mitodl/open-discussions
# fix ocw images

#### Pre-Flight checklist

#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/3526

#### What's this PR do?
This pr fixes thumbnails for ocw next courses in the /learn and /learn/search pages

#### How should this be manually tested?
SET OCW_NEXT_BASE_URL=https://ocwnext-rc.odl.mit.edu/
Run `backpopulate_ocw_next_data` for a few courses if you don't currently have ocw next courses in search locally and in the "New Courses" carousel on the /learn page
Go to http://localhost:8063/learn/ and http://localhost:8063/learn/search. Verify that ocw next courses have thumbnails in the main page and in the expanded resource display.
